assets/src/domain/blocks/event-attendees/attributes.ts

```diff
@@ -29,7 +29,7 @@ const attributes: Block<EventAttendeesAttributes>['attributes'] = {
 	},
 	orderBy: {
 		type: 'string',
-		default: 'lastThenFirstName',
+		default: 'LAST_THEN_FIRST_NAME',
```

Comment:
would be great to have enums for these

Reply:
String literals are even better, they don't require imports and still guard the types.

Reply:
didn't noticed that `Block<EventAttendeesAttributes>['attributes']`, agree this is more convenient